### PR TITLE
[Frontend]: Cart table component(cart total price) 

### PR DESCRIPTION
**Pull request by markjoshuacarmen**
## Ticket/s

- [Cart Checkout Component](https://trello.com/c/pjlstQaT/107-cart-checkout-component)
## Changes

- add cart total price
- checkout button

## Screenshot

- image

## How to test?
- checkout to branch
pnpm run dev
http://localhost:5173/cart

## Notes
- voucher is not included

### CODE REVIEW

File: packages/ui/cart/CartTable.tsx [hunk 1]
```diff
@@ -87,6 +97,39 @@ const CartTable = () => {
...
+              >
+                <Text
+                  fz="sm"
+                  fw={100}
+                  color="black"
+                  style={{ display: 'inline-block' }}
+                >
+                  Total ({totalItems} items):
+                </Text>{' '}
+                &nbsp;&nbsp;
```
Comment (dandalandone) [inline packages/ui/cart/CartTable.tsx]: use margin or padding

File: packages/ui/cart/CartTable.tsx [hunk 2]
```diff
@@ -87,6 +97,39 @@ const CartTable = () => {
...
+                <Text fz="sm" fw={700} style={{ display: 'inline-block' }}>
+                  ₱{formattedSubtotal}
```
Comment (dandalandone) [inline packages/ui/cart/CartTable.tsx]: use html version of peso

File: packages/ui/cart/CartTable.tsx [hunk 3]
```diff
@@ -43,6 +43,16 @@ const CartTable = () => {
   };
 
   const web = useMediaQuery(`(min-width: ${DEFAULT_THEME.breakpoints.sm})`);
+  const totalItems = cartItems.reduce(
+    (total, item) => total + item.quantity,
+    0,
+  );
+  const subtotal = calculateSubtotal(cartItems);
```
Comment (dandalandone) [inline packages/ui/cart/CartTable.tsx:58]: please move computation to back-end

**Review by dandalandone — DISMISSED** [review]
Please see comments, thanks.

File: packages/ui/cart/CartTable.tsx (hunk 1 above)
Comment (markjoshuacarmen) [reply packages/ui/cart/CartTable.tsx]: done

File: packages/ui/cart/CartTable.tsx (hunk 3 above)
Comment (markjoshuacarmen) [reply packages/ui/cart/CartTable.tsx:58]: computation will be move to back-end in api implementation. This will be another ticket

**Review by TreshAnn — APPROVED** [review]
Upon review, I'm satisfied with the changes made to the cart table footer – LGTM! 
 
 please take note of the changes needed in the table body.  Thanks.
 -all computations are handled in the backend.
 -use html version of currency symbol

**Review by ianchinjen — APPROVED** [review]

**Pull request merged**